model.ipynb

```diff
@@ -0,0 +1,433 @@
+{
```

Comment:
<p>Can you save to one csv for all the methods? You can create a dafaframe with cols methodname, date range, p-value and then call <code>df.to_csv(...)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/loglabs/ttb/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Done.